Packages/com.llealloo.audiolink/Runtime/Scripts/ThemeColorController.cs

```diff
@@ -106,11 +152,11 @@ public void OnGUIchange()
             if (!Networking.IsOwner(gameObject))
                 Networking.SetOwner(localPlayer, gameObject);
 #endif
-            customThemeColors[customColorIndex] = Color.HSVToRGB(
+            SetCustomThemeColors(GetCustomThemeColors()[customColorIndex] = Color.HSVToRGB(
```

Comment:
Nitpick, but could you split this into a few statements? This is begging to be misread